src/other/PropertyEditor.js

```diff
@@ -87,6 +87,15 @@
                         return spanText;
                     })
                 ;
+                element.selectAll("i").each(function(){
+                    var i = d3.select(this);
+                    if(i.classed("fa-eye") || i.classed("fa-eye-slash")){
+                        i
+                            .classed("fa-eye",!context.hideNonWidgets())
+                            .classed("fa-eye-slash",context.hideNonWidgets())
+                        ;
+                    }
+                });
```

Comment:
Nothing wrong with this code as such, but feels like you should be able to removw the "each" and the "if" and it would be cleaner (and faster):

Along the lines of (and do you really need to fa-eye selectors?):
```
element.selectAll("i.fa-eye, i.fa-eye-slash")
    .classed("fa-eye",!context.hideNonWidgets())
    .classed("fa-eye-slash",context.hideNonWidgets())
;
```